### ISSUE TITLE
onChange not firing on Input?

### ISSUE THREAD

**Issue by arogan** [Lesha-spr/react-validation#128]
So I have the demo running locally and I'm trying to hook into onChange on an Input and it never fires.  What am I doing wrong?
```
 handleChange = (event) => {
    console.log("onChange = " + event);
  }

  render() {
    return (
      <Form ref={c => { this.form = c }} onSubmit={this.handleSubmit}>
        <div className="row">
          <div className="small-12 columns">
            <h3>Registration</h3>
            <button className="button" type="button" onClick={this.handleClick}>Validate all</button>
          </div>
        </div>
        <div className="row">
          <div className="small-12 medium-6 columns">
            <label>
              Firstname*
              <Input
                placeholder="Firstname"
                type="text"
                name="firstname"
                onChange={this.handleChange}                
                validations={[required]}
              />
            </label>
          </div>
```

**Comment by wladpaiva** [comment]
You are not. The props were being overwritten in the control class.
I made a pull request to fix this bug. You can use my fork for the time being.

**Comment by Lesha-spr** [comment]
Fixed with @3.0.7
